src/js/components/user-management/selfusermanagement.js

```diff
@@ -136,6 +147,24 @@ export const SelfUserManagement = ({ canHave2FA, currentUser, editUser, hasTrack
           </p>
         </div>
       )}
+      <div className="flexbox space-between" style={{ alignItems: 'flex-start' }}>
+        <h2>Locale settings</h2>
+        <FormControl>
+          <Select
```

Comment:
The h2 here doesn't fit in with the rest of the settings I think, maybe something like the _Help us improve Mender_ setting fits in a bit better?